lib/emitter/event-group.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @class
+ * @name EventGroup
+ *
+ * Helper.
+ * It is extend of event emitter for more comfortable work with it.
+ *
+ * @param {Emitter} events
+ *
+ * @example
+ * function MyClass(shower) {
+ *      this._shower = shower;
+ *
+ *      this._message = "Hello";
+ *      this._showerListeners = null;
+ * }
+ *
+ * MyClass.prototype.setupListeners = function () {
+ *      this._showerListeners = this._shower.events.group()
+ *          .on("next", function () { console.log(this._message); }, this)
+ *          .on("prev", function () { console.log(this._message); }, this);
```

Comment:
Me too, but we don't lint comments and this whole class will be dropped in the next PR.